libpairinteraction/State.h

```diff
@@ -22,12 +22,16 @@
 #include <array>
 #include <string>
 #include <iostream>
+#include <cmath>
 #include <boost/functional/hash.hpp>
 
 #include <boost/serialization/array.hpp>
 #include <boost/serialization/string.hpp>
 
 
+constexpr const std::array<char, 7> momentum2label = {{'S', 'P', 'D', 'F', 'G', 'H', 'I'}};
```

Comment:
What if someone selects l = 7?  Better make this a function with a `switch` and a default case which just returns `'X'` or `'_'` or `' '` to indicate the missing letter.  The current solution will just segfault.